image/README.md

```diff
@@ -1,11 +1,13 @@
 # What is this directory?
 
-Tools for creating new disk images for running PipelineV5 VMs on GCP. The image must include all of the tools that are required by
-the pipeline version being executed, and must be a functional standalone OS. The images we create are based on the Debian image
-provided by Google. There are multiple aspects to this image:
-
-* Our tools or their dependencies use Python, R and Perl. As of the update to Debian 12 the latter two of these are managed
-  via `anaconda`.
+Tools for creating new disk images for running `pipeline5` VMs on GCP. The `pipeline5` application consists of a Java application
```

Comment:
These README edits don't relate to CUPPA 2.1.0. Can we move this to a separate ticket/PR?

Reply:
This was addressing the request for improved documentation in a separate email. I can split them out but maybe more pragmatic just to modify the ticket name.